runtime/oti/JITInterface.hpp

```diff
@@ -192,8 +187,18 @@ class VM_JITInterface
 			jitVTableIndex = 0 - (instruction & 0x00000FFF);
 		}
 #elif defined(J9VM_ARCH_AARCH64)
-		/* TODO: Implement this */
-		assert(0);
+		/* Virtual call instructions
+		 *	...
+		 *	movn x9, vTableOffset
+		 *	ldr x9, [register, x9]
+		 *	blr x9
+		 *	<- return address points here
+		 */
+		UDATA movnInstr = ((U_32*)jitReturnAddress)[-3];
+		UDATA ldrInstr = ((U_32*)jitReturnAddress)[-2];
+		if ((ldrInstr & 0xFFFFF81F) == 0xF8696809 && (movnInstr & 0xFFE0001F) == 0x92800009) {
```

Comment:
Can you add a comment in the code that more clearly articulates what this condition is checking for?

Reply:
Added a comment that explains the condition.